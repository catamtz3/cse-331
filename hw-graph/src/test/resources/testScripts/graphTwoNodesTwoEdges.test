# Create a graph
CreateGraph twoEdges

#Add two nodes to the graph
AddNode twoEdges a
AddNode twoEdges b

#Add one edge between nodes a and b
AddEdge twoEdges a b ab
AddEdge twoEdges b a ba

# Get the nodes of the graph
ListNodes twoEdges

# Get the children of the graph
ListChildren twoEdges a
ListChildren twoEdges b