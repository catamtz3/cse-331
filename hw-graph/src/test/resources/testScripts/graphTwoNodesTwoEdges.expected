# Create a graph
created graph twoEdges

#Add two nodes to the graph
added node a to twoEdges
added node b to twoEdges

#Add one edge between nodes a and b
added edge ab from a to b in twoEdges

#Add one edge between nodes b and a
added edge ba from b to a in twoEdges

# Get the nodes of the graph
twoEdges contains: a b

# Get the children of the graph
the children of a in twoEdges are: b(ab)
the children of b in twoEdges are: a(ba)